modules/ROOT/pages/deploy-to-rtf-cli.adoc

```diff
@@ -16,4 +16,157 @@ include::anypoint-cli::partial$cloudhub2-apps.adoc[tags=commands,leveloffset=+1]
+----
+anypoint-cli-v4 plugins:install anypoint-cli-runtime-mgr-plugin
+----
++
+[start=3]
+. Set your credentials by running these commands:
++
+[source,copy]
+----
+anypoint-cli-v4 conf username <username>
+----
+[source,copy]
+----
+anypoint-cli-v4 conf password <password>
+----
+
+To set other parameters for your space, review the xref:anypoint-cli::auth.adoc[Authentication to the Anypoint Platform CLI] documentation.
+After your CLI space is set, you can deploy and manage Mule applications to Runtime Fabric.
+
+=== List Existing Apps
+
+To list existing apps, run the following command:
+[source,copy]
+----
+anypoint-cli-v4 runtime-mgr:application:list.
+----
+
+=== Deploy a Mule App
+
+To deploy the Mule app, run the following command:
+
+[source,copy]
+----
+runtime-mgr:application:deploy <appName> <targetId> <runtimeVersion> <artifactId>
+----
+
+Using (test-mule-1) version as default version: v1.0.6
+deploying new-app01, the output log would be similar to this :
+
+[source,copy]
+----
+╔═════════════════════╤══════════════════════════════════════════════╗
+║ Name:               │ new-app01                                    ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ App Id:             │ 9eeb4bb1-5ba4-47c8-b987-f1e1c2f68f8c         ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Status:             │ UNDEPLOYED                                   ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Updated:            │ a few seconds ago                            ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Runtime:            │ 4.4.0                                        ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Target:             │ 734024b7-002b-40cd-92d2-b659175d0557         ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Artifact Id :       │ test-mule-1                                  ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Replicas:           │ 1                                            ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Update Strategy     │ rolling                                      ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Specs:              │ e45ab9b9-fe58-4387-9975-2ffcb91742df (DESIRED║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Properties:         │                                              ║
+╟─────────────────────┼──────────────────────────────────────────────╢
+║ Secure Properties:  │                                              ║
+╚═════════════════════╧══════════════════════════════════════════════╝
+----
+
+Additionally, you can add more parameters to deploy your Mule app. If you want your app to have 2 replicas instead of 1, you can write `--replicas 2`  at the end of your script.
```

Comment:
```suggestion
Additionally, you can add more parameters to deploy your Mule app. If you want your app to have two replicas instead of 1, you can write `--replicas 2`  at the end of your script.
```